source/installation/resource-manager/kubernetes.rst

```diff
@@ -91,12 +91,66 @@ cluster configuration above or specify them for each app individually.
 
 You should have policies in place to enforce these.
 
-Bootstrapping an Open OnDemand Kuberenetes cluster
-**************************************************
+Bootstrapping the Kuberenetes cluster
```

Comment:
I felt this necessary because don't want people to think the Kubernetes cluster can only do OnDemand things, OnDemand activity can very easily be shared with other Kubernetes workloads.

Reply:
works for me